### ISSUE TITLE
Ao clicar nas estatísticas de um item pela primeira vez é enviado para a página de usuário

### ISSUE THREAD

**Issue by miltonshintaku** [EDUCACAO-LIVRE/DSpace#34]
Pessoal, 
ao clicar a primeira vez na estatística vai para a página de usuário, depois clicando novamente funciona bem

**Comment by miltonshintaku** [comment]
Roddrigo, colocar as estatísticas para serem públicas.... Configuração

**Comment by mfnunes** [comment]
Sim, não esta apresentando as estatísticas na primeira vez, porque o sistema está redirecionando para o login.
